buildSrc/build.gradle.kts

```diff
@@ -77,13 +83,23 @@ gradlePlugin {
             id = "transitiveDependencies" // the alias
             implementationClass = "com.datadog.gradle.plugin.transdeps.TransitiveDependenciesPlugin"
         }
+        register("datadogInstrumentation") {
+            id = "datadogInstrumentation"
+            implementationClass = "com.datadog.gradle.plugin.instrumentation.InstrumentationPlugin"
+        }
     }
 }
 
 tasks.withType<KotlinCompile> {
     kotlinOptions.jvmTarget = JavaVersion.VERSION_11.toString()
 }
 
+tasks.withType<GroovyCompile>().configureEach {
```

Comment:
@mariusc83 just chiming in quickly, since I've built it - you don't need this block for instrumentation to work, it's only needed if you compile groovy classes (in our case was for dexguard support), but I don't see any groovy classes here

Reply:
yes, was just a leftover as I was trying to test several approaches. Thanks for your input.